pagopa/io-app
# feat: [#177760905] Feature logout from identification modal

## Short description
Added the possibility to logout from identification modal.

## How to test
Try to unlock the app, at the bottom of the screen you will see a new label like this "Not profileName?", if you press this label a confirmation popup will be shown, if you press "Continue" your session will be terminated.

**Before changes:**
<img width="300" alt="Screenshot 2021-04-27 at 11 08 01" src="https://user-images.githubusercontent.com/61834253/116217245-bcfbc580-a749-11eb-8744-47a13d62e948.png">

**After changes:**
<img width="300" alt="Screenshot 2021-04-27 at 11 15 16" src="https://user-images.githubusercontent.com/61834253/116217416-e9afdd00-a749-11eb-922e-e0743cc3b96f.png">

<img width="300" alt="Screenshot 2021-04-27 at 12 01 25" src="https://user-images.githubusercontent.com/61834253/116224059-7cec1100-a750-11eb-8379-0ccb4ddc8e17.png">

<img width="300" alt="Screenshot 2021-04-27 at 11 16 13" src="https://user-images.githubusercontent.com/61834253/116217542-09470580-a74a-11eb-9462-4bc20707a311.png">

For some reason the profile name could be undefined, to prevent any error I will show "Logout" instead of "Not profileName?" or "You can login with your SPID or CIE credentials. This session will be disconnected." instead of "You can login with your SPID or CIE credentials. "profileName's" session will be disconnected."
